cri/v1alpha1/cri.go

```diff
@@ -550,6 +550,16 @@ func (c *CriManager) CreateContainer(ctx context.Context, r *runtime.CreateConta
 
 	containerID := createResp.ID
 
+	defer func() {
```

Comment:
Do we need to put this cleanup in daemon?

Reply:
It is easier to handle for cri manager, in addition, it also guarantees the atomicity of the function `Create`. WDYT?

Reply:
What will daemon do if create container fail? do a container data leaving?

Reply:
@Starnop , Yes, I just wonder if we not create through cri interface...

Reply:
> What will daemon do if create container fail? do a container data leaving?

Yeah, just leave it. While for an upper caller, I think if there is an API to provide cleanup action, then call the clean it up.

Reply:
Sure, make sense, still wonder who do this caller if not cri interface.